docs/intro/tutorial02.txt

```diff
@@ -154,9 +154,11 @@ A :class:`~django.db.models.Field` can also have various optional arguments; in
 this case, we've set the :attr:`~django.db.models.Field.default` value of
 ``votes`` to 0.
 
-Finally, note a relationship is defined, using
-:class:`~django.db.models.ForeignKey`. That tells Django each ``Choice`` is
-related to a single ``Question``. Django supports all the common database
+Finally, note that a :class:`~django.db.models.ForeignKey` defines a
+relationship between models. This creates a one-to-many relationship, where
+a ``Question`` can have multiple related ``Choice`` instances. Django supports
+all the common database relationships: many-to-one, many-to-many, and
+one-to-one.
```

Comment:
There's a small problem, when we look at the FK relationship from the target side, it becomes a one-to-many as we have, but FKs are technically many-to-one. Maybe we can look at it from the source side (``Choice``) and keep technical correctness:

```suggestion
relationship between models. This creates a many-to-one relationship, where
each ``Choice`` is related to a single ``Question``, but a ``Question`` can
have multiple ``Choice`` instances associated with it. Django supports all
the common database relationships: many-to-one, many-to-many, and one-to-one.
```

Reply:
LMKWYT

Reply:
That's true, and it would be more consistent with what it says about all the possible database relationships. What if we try "each Choice points to a single Question, but a Question can have multiple Choice instances pointing to it." so that it's clear that it's not the Question pointing to multiple Choice instances? This is very minor though

Reply:
That should be fine 👍🏾

Reply:
With this I don't think there should be any more misinterpretations, how about you?